CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 
-## vNext
+## v0.24.0
 
 - Update Status Lambda - budget_check: Terminate lease if spend > Principal budget amount
+- Add a new `Orphaned` status to accounts (#93)
```

Comment:
We hadn't called it out.  While we created the status there is no current process to put an account in that state.

Reply:
got it.